apps/nuxt3-ssr/components/content/cohort/GeneralDesign.vue

```diff
@@ -104,6 +104,10 @@ function setData() {
       content: mainMedicalCondition,
       type: "ONTOLOGY",
     },
+    {
+      label: "PID",
```

Comment:
Is this the position they like to see it? I would have expected it to be somewhat first?